### ISSUE TITLE
Lottie Player makes 2 network requests to hosted json source

### ISSUE THREAD

**Issue by jj-plane** [LottieFiles/lottie-player#90]
If you take the following example from the Lottie webplayer page

```
  <script src="https://unpkg.com/@lottiefiles/lottie-player@latest/dist/lottie-player.js"></script>
<lottie-player src="https://assets9.lottiefiles.com/packages/lf20_PBgNop.json"  background="transparent"  speed="1"  style="width: 300px; height: 300px;"  loop controls autoplay></lottie-player>
```

If you open the network tab, you can see the hosted JSON asset being called twice, 1 time by xhr and 1 time by fetch.
![fetchandxhrrequest](https://user-images.githubusercontent.com/9252685/130664036-f33aec7e-1b6e-4741-b160-064d1b0f3f84.PNG)

Is there a reason why this happens? Doesn't seem to be an issue when the JSON file is small but what about larger files?

**Comment by karamalie** [comment]
Hi @jj-plane , 
one call is by the wrapper and the other by the underlying Lottie web engine, good find!
Testing a fix and will work it into the next release

**Comment by karamalie** [comment]
Pushed up a fix on v1.1.2 @jj-plane
